build.sbt

```diff
@@ -139,6 +139,17 @@ lazy val core = crossProject.crossType(CrossType.Pure)
 lazy val coreJVM = core.jvm
 lazy val coreJS = core.js
 
+lazy val io = crossProject.crossType(CrossType.Pure)
+  .dependsOn(core, tests % "test-internal -> test")
+  .settings(moduleName := "cats-io")
+  .settings(catsSettings: _*)
+  .jsSettings(commonJsSettings: _*)
+  .jsSettings()
+  .jvmSettings(commonJvmSettings: _*)
+
+lazy val ioJVM = io.jvm
+lazy val ioJS = io.js
+
```

Comment:
I think the new module needs adding to the command aliases (`buildJVM` / `validateJVM`) below & `scripts/travis_publish.sh`